scss/_pagination.scss

```diff
@@ -136,15 +132,16 @@
   }
 
   &:first-child .page-link {
+    margin-right: $pagination-margin-start;
+
     &:not(:empty) {
       padding-right: $pagination-padding-end;
     }
 
-    &::before {
-      /* rtl:raw:
+    /* &::before {
```

Comment:
IMO, you should keep it as is since `/* rtl:raw */` is there to complete rtl version of Boosted.

ATM in rtl : 
![image](https://user-images.githubusercontent.com/91960143/160113329-671300b6-5822-47c2-88ed-1dfd9851de52.png)